src/com/intellij/idea/plugin/hybris/project/descriptors/DefaultHybrisProjectDescriptor.java

```diff
@@ -723,14 +727,23 @@ private void findModuleRoots(
             }
         }
 
-        scanSubdirectories(
-            moduleRootMap,
-            excludedFromScanning,
-            acceptOnlyHybrisModules,
-            rootProjectDirectory.toPath(),
-            progressListenerProcessor
-        );
+        scanForSubdirectories(moduleRootMap, excludedFromScanning, acceptOnlyHybrisModules, rootProjectDirectory.toPath(), progressListenerProcessor);
+    }
 
+    private void scanForSubdirectories(final Map<DIRECTORY_TYPE, Set<File>> moduleRootMap, final Set<File> excludedFromScanning, final boolean acceptOnlyHybrisModules, final Path rootProjectDirectory, final TaskProgressProcessor<File> progressListenerProcessor) {
```

Comment:
both exceptions should be caught by `DefaultHybrisProjectDescriptor.setRootDirectoryAndScanForModules`, can you please comment the reason to catch them here?

Reply:
Yeah you're right I should not handle exceptions here, muscle memory I think,
Try Catch blocks cleaned and signatures added to scanForSubdirectories